### PR TITLE
Clean up travis.yml

### PR DESCRIPTION
**Pull request by NeQuissimus**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,43 +1,39 @@
...
+- export PATH=${PATH}:./vendor/bundle
+
 install:
 - rvm use 2.4.4 --install --fuzzy
 - gem update --system
 - gem install sass
-- gem install jekyll -v 3.8.2
+- gem install jekyll -v 3.8.3
 
 stages:
```
Comment (sideeffffect) [inline .travis.yml]: the less stages, the more parallelism
and we want parallelism :)

File: .travis.yml [hunk 2]
```diff
@@ -1,43 +1,39 @@
...
+- name: lint
```
Comment (sideeffffect) [inline .travis.yml]: the less stages, the more parallelism
and we want parallelism :)

Comment (NeQuissimus) [reply .travis.yml]: yes but I also want a maintainable build definition. Let me mock around with this a bit, I'll see if I can figure this out..

**Comment by jdegoes** [comment]
@NeQuissimus This ready for review / merge?

**Comment by NeQuissimus** [comment]
@jdegoes Yes it is

**Comment by jdegoes** [comment]
@NeQuissimus Thanks for cleaning this up!

File: .travis.yml [hunk 3]
```diff
@@ -1,43 +1,55 @@
...
 - name: test
+- name: microsite
 - name: release
-  if: (branch = master AND type = push) OR (tag IS present)
+  if: (branch = master AND type = push) OR (tag = true)
```
[the remark this answers is not in the document]
Comment (NeQuissimus) [reply .travis.yml]: https://docs.travis-ci.com/user/conditions-v1#boolean-operators

Seems the Travis docs contradict themselves (or both is intended to work)

**Pull request merged**